### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: __unported__/product_images/__openerp__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "version" : "0.2",
     "author" : "Sharoon Thomas, Open Labs Business Solutions, Akretion",
     "website" : "http://openlabs.co.in/",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/product_images/__openerp__.py]: GPL

File: __unported__/product_qt/__openerp__.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "name" : "Products & Pricelists - Define quality control and testing parameters in product",
     "version" : "1.0",
     "author" : "Tiny",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/product_qt/__openerp__.py]: GPL

File: __unported__/product_with_supplier_unit_price/__openerp__.py [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "version" : "0.1",
     "author" : "Savoir-faire Linux",
     "website" : "http://www.savoirfairelinux.com",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/product_with_supplier_unit_price/__openerp__.py]: to remove

Didn't find license as there is a space before `:`

File: product_m2mcategories/__openerp__.py [hunk 4]
```diff
@@ -21,6 +21,7 @@
     "author": "Sharoon Thomas",
     "contributors": ["Mikel Arregi <mikelarregi@avanzosc.es>"],
     "website": "",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline product_m2mcategories/__openerp__.py]: GPL

File: __unported__/product_images/__openerp__.py [hunk 5]
```diff
@@ -23,6 +23,7 @@
     "version" : "0.2",
     "author" : "Sharoon Thomas, Open Labs Business Solutions, Akretion",
     "website" : "http://openlabs.co.in/",
+    "license": "GPL-2 or any later version",
```
Comment (pedrobaeza) [inline __unported__/product_images/__openerp__.py:26]: I'm rewriting this one, so I'll relicense to AGPL-3

**Comment by pedrobaeza** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**